docs/cli/interactive.md

```
@@ -0,0 +1,383 @@
+
+```bash
+# Launch interactive mode (default when no arguments)
+galaxy
+
+# Or explicitly
+galaxy interactive
+```
+
+### Example Session
+
+```
```

markdownlint-cli2 (0.20.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)